src/TerminalFighter/main.cpp

```diff
@@ -103,14 +100,16 @@ void close() {
 }
 
 int main(int argc, char* argv[]) {
+    FLAGS_log_dir = "/tmp/TerminalFighter";
```

Comment:
I kept getting a an error:
`COULD NOT CREATE A LOGGINGFILE 20170816-121402.8765!Could not create logging file: No such file or directory`
until i created `/TerminalFighter` within my `/tmp` directory. 

I think the directory should be created automatically, either by the setup/installer scripts, or have the game check if it exists or something